nodewatcher/nodewatcher.py

```diff
@@ -142,6 +132,22 @@ def maintainSize(region, asg):
         log.debug('capacity less then or equal to min size.')
         return True
 
+
+def saveIdleTime(persisted_data):
+    try:
+        if not os.path.exists('/var/run/nodewatcher/'):
+            os.makedirs('/var/run/nodewatcher/')
+    except OSError as ex:
+        if ex.errno == errno.EEXIST and os.path.exists('/var/run/nodewatcher/'):
+            pass
+        else:
+            log.critical('Persisting idle time failed with exception: %s' % ex)
```

Comment:
I would add contextual info in the log file, something like
'Error persisting idle time data "%s" to file "%s": %s' % (persisted_data, _NW_IDLETIME_FILE, ex)

Reply:
Okay